configure.ac

```diff
@@ -149,34 +149,38 @@ for depfilepath in $list; do
 
     # Get the basename of the file
     depfilename=`basename $depfilepath`
-   
-    # Strip off the .m4 from the filename 
+
+    # Strip off the .m4 from the filename
     depfile=${depfilename%$suffix}
 
     # Extract the subname from the depfile string name
     depsubname=$depfile
     depsubname=${depsubname#$prefix}
-    
+
     happytest=${depfile}_happy
-    
+
     if test "x${!happytest}" = "xyes" ; then
         printf "%38s : YES\n" $depsubname
     else
         printf "%38s : No\n" $depsubname
     fi
-    
+
 done
 
 echo ""
-echo "-------------------------------------------------------" 
+echo "-------------------------------------------------------"
 echo "Configuration Information (Make will build the following elements):"
 echo ""
 
 for e in $dist_element_libraries
 do
     echo "$active_element_libraries" | grep "$e" >/dev/null
     if test $? -eq 0 ; then
-        msg="YES"
+        if test -e "./src/sst/elements/$e/.depreciated" ; then
+            msg="YES - DEPRECIATED"
```

Comment:
deprecated is misspelled

Reply:
You know I thought that the spelling looked funny.